forge-gui/src/main/java/forge/itemmanager/GroupDef.java

```diff
@@ -106,7 +106,7 @@ public Integer apply(final InventoryItem item) {
             }),
 
     CARD_TYPE("lblType",
-            new String[] { "Planeswalker", "Creature", "Sorcery", "Instant", "Artifact", "Enchantment", "Land", "Tribal instant", "Battle" },
+            new String[] { "Planeswalker", "Creature", "Sorcery", "Instant", "Artifact", "Enchantment", "Land", "Battle" },
```

Comment:
is this only used for grouping or also filtering?
wouldn't it be better to keep it as Kindred... 🤔

Reply:
Practically applied in Forge, as far as I can tell, it only shows up as a category when the "group by:" dropdown is set to "Type". 
Conceptually, I'd think it can be used for both in an "at a glance" way really. The thing is I'm not sure it's that interesting to have the cards that are also kindred set apart from the other types they are. My reckoning is, more often than not, if someone is searching for anything kindred-associated it's the creature types.

Reply:
hmn good point :)